EIPS/eip-5192.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 5192
+title: Minimal Soulbound NFTs
+description: Minimal standard interface for soulbinding EIP-721 NFTs
+author: Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-5192-minimal-soulbound-nfts/9814
+status: Review
+type: Standards Track
+category: ERC
+created: 2022-07-01
+requires: 165, 721
+---
+
+## Abstract
+
+This standard is an extension of [EIP-721](./eip-721.md). It proposes a minimal interface to make tokens soulbound using the feature detection functionality of [EIP-165](./eip-165.md).
```

Comment:
Each EIP should be understandable with only its own text, and the EIPs listed in its `requires:` header.

I'd recommend copying over the definition of a soulbound token from your earlier EIP:

```suggestion
This standard is an extension of [EIP-721](./eip-721.md). It proposes a minimal interface to make tokens soulbound using the feature detection functionality of [EIP-165](./eip-165.md). A soulbound token is a non-fungible token bound to a single account.
```